.github/workflows/dev.yaml

```diff
@@ -228,7 +228,7 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '1.19'
-      - uses: azure/setup-kubectl@v1
+      - uses: azure/setup-kubectl@v3.0
```

Comment:
Why use `v3.0` here and not just `v3`? I would assume that the latter would auto-upgrade the minor version.

Reply:
This has been moved to the file [.github/actions/install-dependencies/action.yaml](https://github.com/telepresenceio/telepresence/pull/2838/files/e58897d228406e995040de443ab24feaba2f9ebf..81bd18ef24f405e2c209258d4b87f54d73088d83#diff-342d59190b4737ee45e2062eb625ada477bcea5b4a843b25900ad55d7982f200). I updated that instance.